pages/templates/pages/about_us.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'base.html' %}
+
+{% block content%}
+<h1> TRAVEL AGENCY </h1>
```

Comment:
Maybe it is better if it is in title case

Reply:
done